mediamonks/rest-api-bundle/3.0/config/packages/mediamonks_rest_api.yaml

```diff
@@ -0,0 +1,20 @@
+mediamonks_rest_api:
+    request_matcher:
+        path: /api
+
+    # manually force debug to be enabled or disabled
+    #debug: %kernel.debug%
```

Comment:
you should not put all options commented. The recipe is not meant to replace your documentation, as this file won't be updated with new options when the bundle gets updated (showing the wrong options)